clients/src/main/java/org/apache/kafka/clients/consumer/internals/RequestFuture.java

```diff
@@ -83,15 +87,17 @@ public boolean failed() {
      * @return true if it is retriable, false otherwise
      */
     public boolean isRetriable() {
-        return exception instanceof RetriableException;
+        return exception() instanceof RetriableException;
```

Comment:
this used to not ever throw an exception, and now it can through the `exception()` call. 

idk if that matters or not though